packages/stark-ui/assets/styles/_base.scss

```diff
@@ -26,6 +26,12 @@
   width: 100%;
 }
 
+.stark-app-logout-button {
```

Comment:
Why these CSS rules are in the `_base.scss` file? Shouldn't they be in the `app-logout.component.scss` file ?

Reply:
Actually, this css code can be removed because it duplicates `stark-header-extra` class